test/testlib/minikube_utilities.go

```diff
@@ -756,6 +788,35 @@ func GetAdminEventLog(t *testing.T, namespace string, podName string) {
 	)
 }
 
+func GetFile(t *testing.T, namespace string, podname string, path string, filename string) error {
```

Comment:
is this for some logs that are not published by k8s? And you have to retrieve them from disk?

Reply:
Yes, and Yes.

I am happy to discuss moving this logging to the AP layer, and hence the AP-0 logfile.

For my immediate diagnostics, having a separate file made it quicker to understand the behaviour.